src/main/java/com/revature/models/SwaggerSummary.java

```diff
@@ -26,7 +26,7 @@
     private int threads;
     private int rampUp;
     private boolean followRedirects;
-    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
Is there a reason to prefer lazy loading in this scenario? Since we send back the SwaggerSummary as it is, the application will pretty much always need to fetch this data regardless.

Reply:
We have an endpoint in the SwaggerfileController that retrieves a list of all SwaggerSummaries, so lazy loading speeds up this process significantly.